packages/react-composites/src/composites/common/RichTextSendBoxWrapper.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(rich-text-editor) */
```

Comment:
Just curious if it is needed because of the linter

Reply:
Yes, it was required for linting. As it will flag as unused for stable.